client/src/components/productDetails/DefaultGallery.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import MainImage from './MainImage.jsx';
+import ImageThumbnails from './ImageThumbnails.jsx';
+import ArrowButtons from './ArrowButtons.jsx';
+import GalleryList from './GalleryList.jsx';
+
+const { useState, useEffect } = React;
```

Comment:
do you prefer to extract it or import it? maybe keep it consistent as you also imported in the previous file.